src/components/editor/InputList.jsx

```diff
@@ -49,22 +49,30 @@ class
               .then(json => {
                 for(var i in json){
                   try{
-                    opts.push(json[i]["http://www.loc.gov/mads/rdf/v1#authoritativeLabel"][0]["@value"])
+                    // warns about security/detect-object-injection, but is safe because user cannot inject anything for `var i`
```

Comment:
I was able to suppress these warnings using the following:
```              
                    const item = Object.getOwnPropertyDescription(json, i)
                    const uri = item["@id"]
                    const label = item["http://www.loc.gov/mads/rdf/v1#authoritativeLabel"][0]["@value"]```

Reply:
😎 🆒 

Made this change....